tests/config/pytest_modules.yml

```
@@ -1757,6 +1757,14 @@ ultra/pipeline:
   - modules/ultra/pipeline/**
   - tests/modules/ultra/pipeline/**
 
+umitools/dedup:
+  - modules/umitools/dedup/**
```

```suggestion
  - modules/umitools/dedup/**
  - modules/umitools/extract/**
```

To catch regressions.